spec/javascripts/data_sharing_consent_spec.js

```diff
@@ -27,51 +25,32 @@ describe("Data sharing consent page", function () {
 
     describe("modal confirmation div", function () {
         it("is not shown when the page is loaded", function () {
-            expect($("#consent-confirmation-modal")).toBeHidden();
-        });
-        it("is shown when the failure link is clicked", function () {
```

Comment:
I assume we're removing these specs because they're handled by Bootstrap; shouldn't we still want to ensure the behavior, though?

Reply:
When we use JS features handled by Bootstrap like Bootstrap's `fade` animation on modals, it's hard to test these things exactly, because they don't always update the values we're looking for immediately. There probably is some way to test these things safely, but I'm not too knowledgeable about it right now, and I also don't think we should bother verifying that Bootstrap JS works -- I'd leave it to Bootstrap to verify that (same idea applies for 3rd party Python libs; we wouldn't bother verifying they behave correctly, unless there's some really obvious upstream issue we need to work around).